.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Expected Behavior
+
+### Actual Behavior
+
+### Steps to Reproduce
+
+### Minimal yet Complete Reproducer Code (or URL to Code)
```

Comment:
To keep it consistent with the Netty wiki I would remove caps from every word even if it's a section header / title.